tests/test_other.py

```diff
@@ -5559,11 +5561,6 @@ class Descriptor
     Popen([PYTHON, EMCC, 'src.cpp', '-O2', '-s', 'EXPORT_ALL=1']).communicate()
     assert os.path.exists('a.out.js')
 
-  def test_f0(self):
```

Comment:
this test code relied on names not being minified, and is not important anyhow - it tests an aspect of asm.js float32 support, which has unclear status and really not important given wasm.